[2021-12-24 11:57:19,574] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2021-12-24T11:51:24.100575+00:00 [queued]>
[2021-12-24 11:57:19,609] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.apply_model 2021-12-24T11:51:24.100575+00:00 [queued]>
[2021-12-24 11:57:19,609] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-12-24 11:57:19,610] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-12-24 11:57:19,610] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-12-24 11:57:19,627] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): apply_model> on 2021-12-24T11:51:24.100575+00:00
[2021-12-24 11:57:19,638] {standard_task_runner.py:52} INFO - Started process 1338 to run task
[2021-12-24 11:57:19,647] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'apply_model', '2021-12-24T11:51:24.100575+00:00', '--job-id', '168', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpr3rge4cp', '--error-file', '/tmp/tmpldlo2ch9']
[2021-12-24 11:57:19,661] {standard_task_runner.py:77} INFO - Job 168: Subtask apply_model
[2021-12-24 11:57:19,756] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.apply_model 2021-12-24T11:51:24.100575+00:00 [running]> on host 0d76f33c6135
[2021-12-24 11:57:19,835] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=apply_model
AIRFLOW_CTX_EXECUTION_DATE=2021-12-24T11:51:24.100575+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-24T11:51:24.100575+00:00
[2021-12-24 11:57:56,913] {python.py:118} INFO - Done. Returned value was: None
[2021-12-24 11:57:57,066] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=apply_model, execution_date=20211224T115124, start_date=20211224T115719, end_date=20211224T115757
[2021-12-24 11:57:57,290] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-12-24 11:57:57,422] {local_task_job.py:146} INFO - Task exited with return code 0
